.gitignore

```diff
@@ -1,4 +1,5 @@
 __init__.pyc
+.env
```

Comment:
I think we do need `.env` on version control, which means, **not** in the `.gitignore`

Reply:
Gotcha -- so was wondering the best way to do this if we want .env in version control but I don't want anyone to edit their own .env with the password and then push to the repo....I guess that's something thought that we would just expect to catch on  review?

Reply:
Aha, gotcha. Yes, we avoid that by blocking the `main` branch and requiring a review of every PR.

Reply:
I don't think there is anything like a "read-only" file in version control, which would be the solution to this. Just making it more difficult for destructive commits to get in the codebase :)

Reply:
Perfect! Will make the change.